homeassistant/components/sensor/time_date.py

```diff
@@ -0,0 +1,96 @@
+"""
+homeassistant.components.sensor.time_date
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Date and Time service.
+
+Configuration:
+
+To use the Date and Time sensor you will need to add something like the
+following to your config/configuration.yaml
+
+sensor:
+  platform: time_date
+  monitored_variables:
+    - type: 'time'
+    - type: 'date'
+    - type: 'datetime'
+
+VARIABLES:
+
+monitored_variables
+*Required
+An array specifying the variables to monitor.
+
+These are the variables for the monitored_variables array:
+
+type
+*Required
+The variable you wish to display, see the configuration example above for a
+list of all available variables
+
+
+"""
+import logging
+
+import homeassistant.util.dt as dt_util
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+SENSOR_TYPES = {
+    'time': ['Time'],
```

Comment:
If you remove the square brackets you can refer to it later without the `[0]` index